### PR TITLE
Boundary documentation: content cleanup in prep for devdot migration

### PR DESCRIPTION
**Pull request by taoism4504**
In preparation to migrate boundary docs to devdot in August, I'm taking the liberty of cleaning up some glaring content issues before the migration.

:mag: [Deploy Preview](https://boundary-git-docs-update-learn-to-tutorial-hashicorp.vercel.app/docs)

### CODE REVIEW

**Review by yhyakuna — APPROVED** [review]
LGTM

File: website/content/docs/concepts/domain-model/groups.mdx [hunk 1]
```diff
@@ -51,3 +51,7 @@ A group has the following configurable attributes:
 The following services are relevant to this resource:
 
 - [Group Service](/api-docs/group-service)
+
+## Tutorial
+
+Refer to the [Manage Users and Groups with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-users-groups?in=boundary/hcp-administration) tutorial to learn how to complete user managemement related tasks.
```
Comment (stellarsquall) [inline website/content/docs/concepts/domain-model/groups.mdx:57]: for all tutorial links, I like to remove the collection URL in order to ensure the tutorial can be found if it's moved to another collection, or if a redirect is created. I suffered this recently 

```suggestion

## Tutorial

Refer to the [Manage Users and Groups with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-users-groups) tutorial to learn how to complete user management related tasks.
```
```suggestion

## Tutorial

Refer to the [Manage Users and Groups with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-users-groups?in=boundary/hcp-administration) tutorial to learn how to complete user managemement related tasks.
```

File: website/content/docs/concepts/domain-model/roles.mdx [hunk 2]
```diff
@@ -46,3 +46,7 @@ A role has the following configurable attributes:
 The following services are relevant to this resource:
 
 - [Role Service](/api-docs/role-service)
+
+## Tutorial
+
+Refer to the [Managed Roles and Permissions with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-roles?in=boundary/hcp-administration) tutorial to learn how to create a role in Boundary.
```
Comment (stellarsquall) [inline website/content/docs/concepts/domain-model/roles.mdx:52]:
```suggestion
Refer to the [Managed Roles and Permissions with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-roles) tutorial to learn how to create a role in Boundary.
```
```suggestion
Refer to the [Managed Roles and Permissions with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-roles?in=boundary/hcp-administration) tutorial to learn how to create a role in Boundary.
```

File: website/content/docs/concepts/domain-model/scopes.mdx [hunk 3]
```diff
@@ -84,3 +84,7 @@ A scope has the following configurable attributes:
 The following services are relevant to this resource:
 
 - [Scope Service](/api-docs/scope-service)
+
+## Tutorial
+
+Refer to the [Manage Scopes with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-scopes?in=boundary/hcp-administration) tutorial to learn how to create an org scope and a project scope.
```
Comment (stellarsquall) [inline website/content/docs/concepts/domain-model/scopes.mdx:90]:
```suggestion
Refer to the [Manage Scopes with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-scopes) tutorial to learn how to create an org scope and a project scope.
```
```suggestion
Refer to the [Manage Scopes with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-scopes?in=boundary/hcp-administration) tutorial to learn how to create an org scope and a project scope.
```

File: website/content/docs/concepts/domain-model/sessions.mdx [hunk 4]
```diff
@@ -102,3 +102,7 @@ Changes to a user's permissions do not effect existing sessions.
 The following services are relevant to this resource:
 
 - [Session Service](/api-docs/session-service)
+
+## Tutorial
+
+Refer to the[] Manage Sessions with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-sessions?in=boundary/hcp-administration) tutorial to learn how to start, view, and cancel a session.
```
Comment (stellarsquall) [inline website/content/docs/concepts/domain-model/sessions.mdx:108]:
```suggestion
Refer to the[] Manage Sessions with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-sessions) tutorial to learn how to start, view, and cancel a session.
```
```suggestion
Refer to the[] Manage Sessions with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-sessions?in=boundary/hcp-administration) tutorial to learn how to start, view, and cancel a session.
```

File: website/content/docs/configuration/worker/pki-worker.mdx [hunk 5]
```diff
@@ -57,3 +57,7 @@ worker {
 
 ```
 [kms workers]: /docs/configuration/worker/kms-worker
+
+## Tutorial
+
+Refer to the [Self-Managed Worker Registration with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-workers?in=boundary/hcp-administration) tutorial to learn how to register and manage PKI workers.
```
Comment (stellarsquall) [inline website/content/docs/configuration/worker/pki-worker.mdx:63]:
```suggestion
Refer to the [Self-Managed Worker Registration with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-workers) tutorial to learn how to register and manage PKI workers.
```
```suggestion
Refer to the [Self-Managed Worker Registration with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-workers?in=boundary/hcp-administration) tutorial to learn how to register and manage PKI workers.
```

File: website/content/docs/getting-started/deploy-and-login.mdx [hunk 6]
```diff
@@ -185,6 +185,11 @@ To find your environment’s default `auth_method_id`:
 </Tab>
 </Tabs>
 
+## Tutorial
+
+- Refer to the [Create a Boundary Instance on HCP](https://learn.hashicorp.com/tutorials/boundary/hcp-getting-started-create?in=boundary/hcp-getting-started) tutorial to learn how to deploy a Boundary instance.
+- Refer to the [Install Boundary Desktop](https://learn.hashicorp.com/tutorials/boundary/hcp-getting-started-desktop-app?in=boundary/hcp-getting-started) tutorial to learn how to use the Boundary desktop to connect to a running HCP Boundary instance.
```
Comment (stellarsquall) [inline website/content/docs/getting-started/deploy-and-login.mdx:191]:
```suggestion
- Refer to the [Create a Boundary Instance on HCP](https://learn.hashicorp.com/tutorials/boundary/hcp-getting-started-create) tutorial to learn how to deploy a Boundary instance.
- Refer to the [Install Boundary Desktop](https://learn.hashicorp.com/tutorials/boundary/hcp-getting-started-desktop-app) tutorial to learn how to use the Boundary desktop to connect to a running HCP Boundary instance.
```
```suggestion

**Review by stellarsquall — CHANGES_REQUESTED** [review]
looking great! 

only two real comments, one on using direct tutorial links instead of including the collection context, the other was a question about style guide bits.

**Review by stellarsquall — APPROVED** [review]
great work!

**Pull request merged**